docs/components/dropdown-menu.md

```diff
@@ -0,0 +1,394 @@
+# DropdownMenu
+
+**KDropdownMenu** is a button (or any slotted content) that is clicked to trigger a menu popover beneath it.
+
+<KDropdownMenu label="Documentation" :items="deepClone(defaultItemsUnselected)" />
+
+## Props
+
+### label
+
+The label for the menu.
+
+### items
+
+An array of items containing a `label` and other optional properties will render a menu of [`KDropdownItems`](#KDropdownItem) .
+
+```html
+<KDropdownMenu
+  label="Documentation"
+  :items="[
+    { label: 'Home', to: { path: '/' } },
+    { label: 'Button docs', to: { path: '/components/button.html' } },
+    { label: 'My docs', to: { path: '/components/dropdown-menu.html' } }
+  ]"
+/>
+```
+
+### appearance
+
+Use this prop to specify the display style for the dropdown menu. Can be either `menu` (default) or `selectionMenu`.
+The `menu` style is the standard you have seen in the example above.
+
+The `selectionMenu` style is good for a clearer representation of what is selected. `selected` state is handled automatically when
+clicking a `KDropdownItem` if used in conjunction with the `items` prop. You will need to manually control selectedness if you are using the `items` slot.
```

Comment:
"selectedness" can we reword this? to something like this? (Unsure if this reads correctly 😬)
```suggestion
clicking a `KDropdownItem` if used in conjunction with the `items` prop. 

If you instead utilize the `items` slot, you will need to manually control the reactive state of the selected item.
```

Reply:
Fixed (I think yours reads better than mine, I like to make up words 😓)